ci/Jenkinsfiles/release.groovy

```diff
@@ -270,7 +270,7 @@ pipeline {
     success {
       script {
         if (env.DRY_RUN != 'true') {
-          currentBuild.description = "Release nuxeo/nuxeo ${RELEASE_VERSION} from build ${params.BUILD_VERSION}"
+          currentBuild.description = "Release ${RELEASE_VERSION} from build ${params.BUILD_VERSION}"
           slackSend(channel: "${SLACK_CHANNEL}", color: 'good', message: "Successfully released ${RELEASE_VERSION} from build ${params.BUILD_VERSION}: ${BUILD_URL}")
```

Comment:
Don't you want to state "nuxeo/nuxeo" in the slack message still?

Reply:
Phew, yes, you're right!